homeassistant/components/sensibo/services.yaml

```diff
@@ -80,3 +80,69 @@ enable_pure_boost:
           options:
             - "Normal"
             - "Sensitive"
+full_state:
+  name: Set full AC state
+  description: Set full AC state for Sensibo device.
+  target:
+    entity:
+      integration: sensibo
+      domain: climate
+  fields:
+    mode:
+      name: HVAC mode
+      description: HVAC mode to set
+      required: true
+      example: "heat"
+      selector:
+        select:
+          options:
+            - "cool"
+            - "heat"
+            - "fan"
+            - "auto"
+            - "dry"
+            - "off"
+    target_temperature:
+      name: Target Temperature
+      description: Optionally set target temperature
+      required: false
+      example: 23
+      selector:
+        number:
+          min: 0
+          step: 1
+          mode: box
+    fan_mode:
+      name: Fan mode
+      description: Optionally set fan mode
+      required: false
+      example: "low"
+      selector:
+        text:
+          type: text
+    swing_mode:
+      name: swing mode
+      description: Optionally set swing mode
+      required: false
+      example: "fixedBottom"
+      selector:
+        text:
+          type: text
+    horizontal_swing_mode:
+      name: Horizontal swing mode
+      description: Optionally set horizontal swing mode
+      required: false
+      example: "fixedLeft"
+      selector:
+        text:
+          type: text
+    light:
+      name: Light
+      description: Set light on or off.
```

Comment:
```suggestion
      description: Set light on or off
```

Reply:
I'm not sure if we have a consistent standard for ending with a `.` or not, but it would be nice to have it consistent for all descriptions in the same service (or even the whole file).

Reply:
I remove all the `.` here now and I will do the rest in the separate PR